src/interfaces/ConfigInterface.ts

```diff
@@ -0,0 +1,21 @@
+import type { Store } from "pinia";
+
+export interface ConfigStore {
+    api: {
+        botDetector: string;
+        runeLite: string;
+    }
+    starterBotHunter: string[];
+    BronzeBotHunter: string[];
+    IronBotHunter: string[];
+    SteelBotHunter: string[];
+    MithrilBotHunter: string[];
+    AdamantBotHunter: string[];
+    RuneBotHunter: string[];
+    DragonBotHunter: string[];
+}
+
+export type IConfigStore = Store<"configStore", ConfigStore, {}, {
```

Comment:
Hungarian notation does not apply to types, and should not, the notation here should be on the above interface however I recommend not using it at all

Reply:
Why are you not recommending the interface? If I dont use any type for the configStore when using it in the apiStore typescript is throwing errors since it sees configStore as an any variable which it should not be, I there another approach for this?